### DISCUSSION TITLE
Permission - Use Custom - Item Permission all

### DISCUSSION THREAD

**Discussion by NilsBaumgartner1994** [directus/directus#14433] [Feature Requests]
Consider the following. A User has a Device. A user should only be allowed to read own devices. This is perfectly configurable.
You head into Permission-->Use Custom and uses the $CURRENT_USER variable and allows to read all fields.

You consider a field is missing in an entity. You add the missing field. Now you need to head to update all field permissions...
- You will need to go through all Roles
- You will need to go through all Actions (Read, Update, Delete, Create)

This is something I am facing. Often I want the filter rule only, to select **who** is operating. A nice feature would be to set a checkbox (New Fields Default: true/false). 

Otherwise an other feature would be at creating a field in an entity, to set the permissions directly in the Interface:
--> Add new Field
  --> "Hey seems like role X has access to this entity. You want to allow [update, create, delete, read] the new field?"

**Comment by rijkvanzanten** [comment]
This is technically the difference between `*` and manually listing all fields. Even though the UI currently doesn't have that toggle yet, you could already get around this by setting `fields` to `*` through the API's `/permissions` endpoint 🙂

**Reply by NilsBaumgartner1994** [reply]
Yes, technicly that's what I am looking for :-)
But consider my other thought, that only future fields should be allowed. This would allow you to exclude specific fields (like intern notes).

But still this would be something I would like to see in the UI with a toggle 🥇 
Anyways I want to say a big thank you to you guys, for this awesome software and awesome support !

**Reply by rijkvanzanten** [reply]
Ahh interesting. I think that coincides with the "exclude fields from `fields` parameter" request, in that you could set something like `*, -author` to allow everything, but omit the author field 🤔

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
